src/sentry/incidents/subscription_processor.py

```diff
@@ -322,24 +326,12 @@ def get_crash_rate_alert_metrics_aggregation_value(
         count is just ignored
         - `crashed` represents the total sessions or user counts that crashed.
         """
-        row = subscription_update["values"]["data"][0]
-        total_session_count = row.get("count", 0)
-        crash_count = row.get("crashed", 0)
-
-        if total_session_count == 0:
+        # NOTE (mifu67): we create this helper because we also use it in the new detector processing flow
+        aggregation_value = get_crash_rate_alert_metrics_aggregation_value_helper(
+            subscription_update
+        )
+        if aggregation_value is None:
             self.reset_trigger_counts()
```

Comment:
A little hard to tell because of the diff but where is this being called? I think we'd want to remove it. If we've removed all the calls to it we can remove the function too.

Reply:
Where is what being called?